Makefile

```diff
@@ -21,8 +21,23 @@ GOFLAGS ?= $(GOFLAGS:)
 BUILD_TIME := $(shell date -u +%Y%m%d.%H%M%S)
 BUILD_HASH := $(shell git rev-parse HEAD)
 
+################################################################################
+
+AWS_SDK_URL := github.com/aws/aws-sdk-go
+AWS_SDK_VERSION := $(shell find go.mod -type f -exec cat {} + | grep ${AWS_SDK_URL} | cut -c28-)
```

Comment:
This is a tiny nit-pick but can we change the last part of these pipelines to `| awk '{print $NF}'`? It's ever-so-slightly less brittle, because it will always select the last column rather than using -c to count characters from the beginning.

Also, what is the behavior of this if the targetted Go module is not present?